CMakeLists.txt

```diff
@@ -150,6 +151,15 @@ if (Spectra_FOUND)
   list (APPEND OPENTURNS_PRIVATE_LIBRARIES Spectra::Spectra)
 endif()
 
+if (USE_PRIMESIEVE)
+  find_package (primesieve CONFIG)
+  if (primesieve_FOUND)
+    set (OPENTURNS_HAVE_PRIMESIEVE TRUE)
+    message (STATUS "Found primesieve: ${primesieve_DIR} (found version ${primesieve_VERSION})")
+    list (APPEND OPENTURNS_PRIVATE_LIBRARIES primesieve::primesieve)
+  endif ()
+endif ()
+
```

Comment:
unfortunately on ubuntu here they dont package the config.cmake, here is a patch to use a custom macro:
[0001-findps.txt](https://github.com/openturns/openturns/files/7250659/0001-findps.txt)

Reply:
I added primesieve into the linux & mingw images so it will be tested now.

Maybe you could also add it to macos by editing .github/workflows/build.yml, look for the "brew install" command